index.html

```diff
@@ -0,0 +1,135 @@
+    <title>capstone-project</title>
+    <link rel="stylesheet" href="style.css"/>
+    <script src="https://kit.fontawesome.com/f25b74c76b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header id="dsktop-header">
+        <nav id="nav1" title="first-navigation">
+          <ul>
+            <li><a class="fa-brands fa-facebook-f"></a></li>
+            <li><a class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="nav1item">English</a></li>
+            <li><a href="#" class="nav1item">My Page</a></li>
+            <li><a href="#" class="nav1item">Logout</a></li>
+          </ul>
+        </nav>
+         
+        <nav id="nav2" title="second-navigation"> 
+          <a class="main-logo" href="#home"><img src="images/icecream.png" alt="logo"></a>
+          
+          <a class="nav2Item" href="./about.html">About</a>
+          <a class="nav2Item" href="#packages">Packages</a>
+          <a class="nav2Item" href="./about.html">Join</a>
+          <a class="nav2Item" href="#sponsors">Sponsor</a>
+          <a class="nav2Item" href="./about.html">Recipes</a>
+          <a class="mainItem" href="./about.html">Workshops</a> 
+        </nav>   
+    </header>
+
+    <header id="mobile-header">
+      <nav class="mobile-nav"> 
+          <ul onclick="menuPopup()" id="navList" title="navigation-list">
+            <li><a class="navItem" href="./about.html">About</a></li>
+            <li><a class="navItem" href="#packages">Packages</a></li>
+            <li><a class="navItem" href="./about.html">Join</a></li>
+            <li><a class="navItem" href="#sponsors">Sponsor</a></li>
+            <li><a class="navItem" href="./about.html">Recipes</a></li>
+            <li><a class="navItem navItem-main" href="./about.html">Workshops</a></li>
+          </ul> 
+      </nav>   
+      <nav>
+        <div  id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div> 
+        <div  id="closeButton"> &times; </div>
+      </nav>
+    </header>
+
+    <section class="mobilemain">
+        <article class="home-page" id="#home" title="home-page">
+          <h3 class="intro-title-one"><q>cakey-cakey by Marianna</q></h3>
+          <h1 class="intro-title-two">CAKE-LOVERS WORKSHOP FOR MINORITY GROUP 2023</h1>
+            <p class="explanation-paragraph" alt="explanation-paragraph">
+              A workshop that will allows low income individuals to
+              exhibit with low income to get source of income why can sell their cake to the host community and make money out of it</p>
+          <h4 class="event-date" alt="event-date"> 2023.05.25(THU) ~ 26(FRI)</h4>
+          <div class="copyright">&copy; Nairobi, Kenya </div>
+        </article>
+
+        <article id="packages-offered" title="packages-offered">
+            <h5 class="packages-title" id="packages"> Packages Offered</h5>
```

Comment:
- [optional] Great job . like in the original design you can add the width of your cards as they are supposed to be wider than the join us button.
- [optional] Please instead of using lecture in all the cards you can edit them to represent different things. nice work
yours
![Screenshot (1087)](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/65298048/88017059-3c59-4bba-9a9a-c3671ebc953d)
main design
![Screenshot (1086)](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/65298048/42de68ec-6367-457d-9d40-43d22a6018bb)